client/src/components/Spinner.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export default function Spinner() {
+  return (
+    <div className="spinner">
+      <div className="circle">
+        <label>Loding...</label>
```

Comment:
```suggestion
        <p>Loading...</p>
```
Like my other comment, this should not be a label. We could use `p` instead.